README.md

```diff
@@ -1,6 +1,7 @@
 # docsite
 Source Code of the ReactPlay Documentation Website
-
+# Link to the original repo of ReactPlay: 
```

Comment:
Hello @ArslanYM , thanks for the PR,

Can we just put simple text link instead of putting the heading like this?